Week2/homework/Exercise3.js

```diff
@@ -0,0 +1,8 @@
+const fruitBasket = ['Apple', 'Lemon', 'Grapefruit', 'Lemon', 'Banana', 'Watermelon', 'Lemon'];
+
+const newbasket = fruitBasket.filter(funct);
+function funct (item){
+  var extract = item != 'Lemon';  
+  return extract;
```

Comment:
Declaring extract is redundant here. You can do:  
`return item != 'Lemon';` directly